### PR TITLE
Containers org

### LINKED ISSUE #71: Being added to the https://github.com/containers org
Hi all! Charlie from the Podman Desktop team.

I've been in talks with the containers org, and we'd like to have you be apart of it considering the closeness this tool is to podman.

Podlet is amazing at generating the systemd files and it'd be great to have it be apart of the containers organization.

Let me know if you are all on board with this!

### PR DESCRIPTION
**Pull request by k9withabone**
- Added a code of conduct, a security policy, and contribution guidelines.
- Updated links to repository to reflect the new location.
- The podlet container image was moved to ghcr.io/containers/podlet.

Closes #71

### CODE REVIEW

**Comment by k9withabone** [comment]
@cdrage, could you review `CODE-OF-CONDUCT.md`, `CONTRIBUTING.md`, and `SECURITY.md`?

File: Containerfile [hunk 1]
```diff
@@ -31,5 +31,8 @@ RUN cargo build \
 RUN cp target/$(cat /rust_target.txt)/dist/podlet .
 
 FROM scratch
+LABEL org.opencontainers.image.source="https://github.com/containers/podlet"
+LABEL org.opencontainers.image.description="Generate podman quadlet files from a podman command, compose file, or existing object"
```
Comment (TomSweeneyRedHat) [inline Containerfile:35]: Except for Podman commands themselves, we try to keep it Uppercased.

```suggestion
LABEL org.opencontainers.image.description="Generate Podman Quadlet files from a podman command, compose file, or existing object"
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
 ### Features
 
-- Add support for quadlet options introduced in podman v4.7.0 ([#29](https://github.com/k9withabone/podlet/issues/29))
+- Add support for quadlet options introduced in podman v4.7.0 ([#29](https://github.com/containers/podlet/issues/29))
```
Comment (TomSweeneyRedHat) [inline CHANGELOG.md:60]:
```suggestion
- Add support for quadlet options introduced in Podman v4.7.0 ([#29](https://github.com/containers/podlet/issues/29))
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Podlet
 
 ![Crates.io](https://img.shields.io/crates/v/podlet?style=flat-square)
-![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/k9withabone/podlet/ci.yaml?event=push&label=ci&logo=github&style=flat-square)
+![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/containers/podlet/ci.yaml?event=push&label=ci&logo=github&style=flat-square)
 ![Crates.io License](https://img.shields.io/crates/l/podlet?style=flat-square)
 
 Podlet generates [podman](https://podman.io/) [quadlet](https://docs.podman.io/en/stable/markdown/podman-systemd.unit.5.html) files from a podman command, compose file, or existing object.
```
Comment (TomSweeneyRedHat) [inline README.md:7]:
```suggestion
Podlet generates [Podman](https://podman.io/) [Quadlet](https://docs.podman.io/en/stable/markdown/podman-systemd.unit.5.html) files from a Podman command, compose file, or existing object.
```

**Review by TomSweeneyRedHat — COMMENTED** [review]
Nit picking caps, otherwise LGTM

**Comment by cdrage** [comment]
@TomSweeneyRedHat is the best to ask and I've seen he has done a review! 

Above all LGTM after his changes

**Comment by k9withabone** [comment]
Just found out that the `--annotation` option on `podman manifest create` is new to Podman v5.0.0. Looks like I'll need to figure out how to run newer versions of Podman/Buildah (current version in the runner is Podman v4.6.2, from the Kubic project) or a different way to update the manifest for the `release-container` action.

**Comment by k9withabone** [comment]
Thanks for the review @TomSweeneyRedHat! I'll work on fixing the capitalization throughout the documentation.

**Pull request merged**